jdbc/src/main/java/nextstep/jdbc/RowMapper.java

```diff
@@ -0,0 +1,9 @@
+package nextstep.jdbc;
+
+import java.sql.ResultSet;
+
+@FunctionalInterface
+public interface RowMapper<T> {
+
+    T run(final ResultSet rs);
```

Comment:
```suggestion
@FunctionalInterface
public interface RowMapper<T> {

    T run(final ResultSet rs) throws SQLException;
```

RowMapper를 실제로 사용하는 곳은 JdbcTemplate 내부이므로
throws 선언을 통해 라이브러리 사용자가 try/catch를 하지 않아도 되게 해줄 수 있을 것 같아요!

Reply:
아 그러네요! 클라이언트 입장에서는 `RowMapper` 를 생성해서 넘겨주기만 할 뿐이니 여기에 `throws` 를 추가해서 `JdbcTemplate` 에 예외 처리를 미루면 되겠군요!

Reply:
아하 그렇군요 😲 RowMapper 를 생성하는 것은 클라이언트지만, 전달받아 직접 실행하는 측은 JdbcTemplate 라이브러리 이므로, 클라이언트가 직접 예외처리를 하지 않아도 되네요!